docs/remote_inference_blueprints/standard_blueprints/sagemaker_semantic_highlighter_standard_blueprint.md

```diff
@@ -0,0 +1,133 @@
+# AWS SageMaker Semantic Highlighter Model Standard Blueprint
+
+This blueprint demonstrates how to deploy a semantic highlighter model using AWS SageMaker and integrate it with OpenSearch. For a detailed Python-based tutorial on deploying the model to SageMaker, please refer to the [Deploying OpenSearch Sentence Highlighter Model To AWS SageMaker Guide](https://github.com/opensearch-project/opensearch-py-ml/docs/source/examples/aws_sagemaker_sentence_highlighter_model/README.md).
+
+## Overview
+
+The semantic highlighter model helps identify and highlight the most relevant parts of a text passage based on a given question. This blueprint shows how to:
+
+1. Create a SageMaker connector
+2. Register a model group
+3. Register and deploy the model
+4. Test the model inference
+
+## Prerequisites
+
+1. AWS account with SageMaker access
+2. SageMaker endpoint deployed with the semantic highlighter model
+3. AWS credentials with appropriate permissions
+
+## Steps
+
+### 1. Create SageMaker Connector
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "semantic_highlighter_connector",
+  "description": "Connector for semantic highlighter model on SageMaker",
+  "version": "1",
+  "protocol": "aws_sigv4",
+  "parameters": {
+    "region": "<YOUR_AWS_REGION>",
+    "service_name": "sagemaker",
+    "endpoint": "runtime.sagemaker.{region}.amazonaws.com",
+    "model": "{your-endpoint-name}"
+  },
+  "credential": {
```

Comment:
Why not use IAMRole for credential here?

Reply:
Offline synced with @jngz-es. This blueprint is general target to open source OpenSearch cluster, not specific to managed service. We have to use credentials.